ehcache-impl/src/main/java/org/ehcache/config/builders/CacheConfigurationBuilder.java

```diff
@@ -558,7 +558,10 @@ public CacheConfigurationBuilder<K, V> withSizeOfMaxObjectGraph(long size) {
    *
    * @see #withSizeOfMaxObjectGraph(long)
    * @see #withDefaultSizeOfSettings()
+   *
+   * @deprecated Use {@link #withSizeOfMaxObjectGraph(long) } or {@link #withDefaultSizeOfSettings() } instead
```

Comment:
How are the linked methods a replacement for this deprecated one?

Reply:
removed the @deprecated from java doc